.github/workflows/tests.yml

```diff
@@ -8,53 +8,32 @@ on:
 
 name: Tests
 jobs:
-  test:
-    name: Test suite (mainnet, testnet, betanet)
+  test-mainnet:
+    strategy:
+      matrix:
```

Comment:
Keep in mind that we've previously discovered that running tests sequentially gives us better execution time because of better cache hit